packages/botfuel-dialog/src/dialogs/text-dialog.js

```diff
@@ -23,10 +23,10 @@ const Dialog = require('./dialog');
  */
 class TextDialog extends Dialog {
   /** @inheritDoc */
-  async execute(adapter, userMessage, messageEntities) {
-    logger.debug('execute', userMessage, messageEntities);
-    const extraData = await this.dialogWillDisplay(userMessage, messageEntities);
-    const dialogData = { messageEntities, extraData };
+  async execute(adapter, userMessage, data) {
+    logger.debug('execute', userMessage, data);
+    const extraData = await this.dialogWillDisplay(userMessage, data);
+    const dialogData = { data, extraData };
```

Comment:
I would have expected:
const dialogData = { ...data, extraData };
maybe we need to discuss